### PR TITLE
Accept double digit chassis and blade numbers in messages

### PR DESCRIPTION
**Pull request by PakDLiu**
Regex was not allowing the plugin to fetch service profile if chassis number is double digit

### CODE REVIEW

**Review by Gabedlm — APPROVED** [review]

File: lib/fluent/plugin/filter_process_ucs_syslog.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ProcessUcsSyslog < Filter
     @@tagesRegex = /\[[\w]+\]\[([\w]+)\]\[([-\w]+)\]\[([-\/\w]+)\]/
     @@restartAdaptorRegex = /Adapter (\d)\/(\d)\/\d restarted/
 
-    @@bladeRegex = /sys\/chassis-(\d)\/blade-(\d)/
+    @@bladeRegex = /sys\/chassis-([\d]+)\/blade-(\d)/
```
Comment (jovillas) [inline lib/fluent/plugin/filter_process_ucs_syslog.rb]: Shouldn't we accept also cases with more than 9 blades?

Comment (lagalbra) [inline lib/fluent/plugin/filter_process_ucs_syslog.rb]: We should update the blade number part of the regex as well to ward off this same issue there

**Review by lagalbra — CHANGES_REQUESTED** [review]

File: lib/fluent/plugin/filter_process_ucs_syslog.rb (hunk 1 above)
Comment (PakDLiu) [reply lib/fluent/plugin/filter_process_ucs_syslog.rb]: I have not noticed any blade that has double digits. but i'll do it just in case

**Review by jovillas — APPROVED** [review]

File: lib/fluent/plugin/filter_process_ucs_syslog.rb (hunk 1 above)
Comment (PakDLiu) [reply lib/fluent/plugin/filter_process_ucs_syslog.rb]: already done

**Review by lagalbra — CHANGES_REQUESTED** [review]
Add a specific unit test for `test_event_filter_double_digit_blade` like you did `test_event_filter_double_digit_chassis`

**Comment by PakDLiu** [comment]
The test checks for both, i guess i should rename the test

**Comment by PakDLiu** [comment]
Added specific tests

**Review by lagalbra — APPROVED** [review]
Update the PR title to represent both chassis and blade numbers, and you're looking good!

**Pull request merged**